core/tests/revert-test/tests/revert-and-restart.test.ts

```diff
@@ -40,8 +41,16 @@ function parseSuggestedValues(suggestedValuesString: string): {
     };
 }
 
-async function killServerAndWaitForShutdown(tester: Tester) {
-    await utils.exec('killall -9 zksync_server');
+async function killServerAndWaitForShutdown(tester: Tester, serverProcess: ChildProcessWithoutNullStreams) {
```

Comment:
if you want to reuse this function in autokill you have to return back `await utils.exec('killall -9 zksync_server');` for undefined in server process

Reply:
Good catch! Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/e7e4cfb8bcc54a2c58bfaa54c51f0fecbfe44e6a